qcodes/instrument_drivers/AlazarTech/ATS9440.py

```diff
@@ -11,7 +11,7 @@ class AlazarTech_ATS9440(AlazarTech_ATS):
     samples_divisor = 256 #32
 
     def __init__(self, name, **kwargs):
-        dll_path = 'C:\\WINDOWS\\System32\\ATSApi.dll'
+        dll_path = '/usr/lib64/libATSApi.so'
```

Comment:
I think you were trying the driver on Linux. As we want to stay on Windows, you should keep the windows API address.

Reply:
Yea that was just for my personal testing. Changed it back to Windows.